Firmware/mmu2.cpp

```diff
@@ -691,9 +691,11 @@ void MMU2::OnMMUProgressMsg(ProgressCode pc){
                     break;
                 case FilamentState::NOT_PRESENT:
                     // fsensor not triggered, continue moving extruder
-                    current_position[E_AXIS] += 5.0f;
-                    plan_buffer_line_curposXYZE(MMU2_LOAD_TO_NOZZLE_FEED_RATE);
-                    st_synchronize(); // Wait for the steps to be done so the moves don't pile up
+                    if(!blocks_queued())
```

Comment:
in this case, using blocks_queued() can actually work just fine, because we are "playing" just with the extruder (and other axes are standing still)